teaspoons-cli/teaspoons/auth_helper.py

```diff
@@ -42,7 +49,7 @@ def _validate_token(token: str) -> bool:
         # This is just to ensure the token is not expired
         jwt.decode(token, options={"verify_signature": False, "verify_exp": True})
         return True
-    except Exception as e:
+    except Exception:
```

Comment:
I think it could improve error handling to catch more specific exceptions here. Like this:
```
except jwt.ExpiredSignatureError:
    # log message that token is expired
    return False
except Exception as e:
    # log error message from `e` or do something else
    return False
```
This way, if expiration is the only concern, it’s clear, and any other JWT-related issues can be handled as well. My guess is it's still possible that TokenInvalid can be raised here if non-JWT decodable thing is provided. Let me know what you think!